### PR TITLE
Set primary tox mode to serial, add parallel opt

### PR DESCRIPTION
**Pull request by adberglund**
## Summary
Instead of completely disabling the ability to run tests in parallel, let's just reset the primary tox run to be in serial.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -24,6 +24,30 @@ application-import-names = koku, api, reporting, reporting_common, cost_models,
...
+  codecov
+commands =
+  /bin/sh {toxinidir}/scripts/check_postgres_running.sh
+  /bin/sh {toxinidir}/scripts/create_test_db_user.sh
+  pipenv install --dev --ignore-pipfile
+  coverage run {toxinidir}/koku/manage.py test --noinput -v 2 {posargs: koku/}
+  coverage report --show-missing
+
+
+# The same tests as testens, just without parallel
```
Comment (infinitewarp) [inline tox.ini]: I think this comment is backwards. `s/out//`?

Comment (adberglund) [inline tox.ini]:
```suggestion
# The same tests as testenv, but run in parallel
```

**Comment by adberglund** [comment]
@infinitewarp The masu part isn't run as part of travis, and is just a local convenience since masu was originally a separate repo.

**Comment by infinitewarp** [comment]
womp womp… 😖 
https://travis-ci.org/project-koku/koku/jobs/574856976#L5626
```
Error: HTTP 400

Please upload with the Codecov repository upload token to resolve issue.
```

**Pull request merged**